### PR TITLE
A11y fixes

### PR DESCRIPTION
**Pull request by SamPotasz**
Fixes issues #1 #2 #3 #4 and #5

### CODE REVIEW

**Comment by SamPotasz** [comment]
> Fixes issues #1 #2 #3 #4 and #5

Also now fixes #7 and #8

**Review by dianalam — COMMENTED** [review]
thanks! the buttons look good! I noticed that the fixes changed the layout of the homepage (the header and main are now misaligned, "get started" became "find ableist language in your job posting", and the "how to use this tool" moved out of the blue box and into the main body text). is there a reason you made these changes?

image

**Comment by SamPotasz** [comment]
Ah, my bad on the navbar misalignment. Should be fixed with the latest push.

As for the headers, there wasn't an `<h1>` on the page before, and best practice is to have (exactly) one on each page. I did my best for the homepage but please edit it as you see fit!

I moved the explanatory text out of the alert box because to me it made the flow of the document hard to follow, and the way it was laid out it sometimes appeared above the text and sometimes below. Seemed to make sense to just put it as intro text above the form at all times.

Upon typing this out, maybe it makes sense to just have "Getting Started" be the h1, eliminate other subheadings, and keep the explanatory text above the form.

How does that seem?

**Comment by dianalam** [comment]
Thanks! the `h1` recommendation makes sense. I just pushed some edits to try and consolidate the headers: @SamPotasz and @mtpatter: lmk what you think. @SamPotasz - would it be possible to adjust the line spacing under the headers? Seems like there's now too much space under "how to use this tool" and "get started." The font size of "paste the job description..." also seems bigger than the font size of the explanatory paragraph above--could we fix this? 

And one other thing I noticed - the text box on the home page is now much narrower than the one on the results page; can we increase the size of the text box on the home page so that they're consistent?

**Comment by SamPotasz** [comment]
Thanks, @dianalam :-) I just pushed a few changes to fix the spacing, text size, and textarea width issues you mentioned.

I think the formatting should be good to go now. I modified the text of the H1 because the paragraph right below didn't seem to answer the question "How to use this tool," but I'll leave any further content changes up to the team!

**Comment by dianalam** [comment]
@SamPotasz Thanks! Changes look great. 

I noticed one more thing: On the results page, the yellow text box with suggestions now gets appended to after every click instead of being replaced with the information for the word that was clicked on. Did one of the a11y/formatting changes generate this effect?

image

File: static/main.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  ul_end = "</ul>"
+
+  example = `<p>Example: ${words.dataset.example}</p>`
+  let toAdd = header + subtitle + ul_start;
+  for (let i = 0; i < alt_verbs.length; i++) {
+    verb = alt_verbs[i]
+    toAdd += `<li class="margin-bottom-1">${verb}</li>`
+  }
+  toAdd += ul_end + example
+  outputEl.innerHTML = toAdd
```
Comment (SamPotasz) [inline static/main.js:23]: previous line was `+= toAdd` which caused the error

**Comment by dianalam** [comment]
thanks!
